CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 # History
 
+## 3.2.0
+- Widget:
+    - `UploadcareWidget.selectFile()` was deprecated since `Fragment.startActivityForResult` was deprecated.
+      We provide a new way via `Fragment.registerForActivityResult` and `Activity.registerForActivityResult`
```

Comment:
A new way of what? :)

Reply:
A new way of launching selection of the file to upload

Reply:
Maybe add these words to the changelog?